src/vs/workbench/contrib/chat/browser/chat.contribution.ts

```diff
@@ -125,8 +127,18 @@ class ChatAccessibleViewContribution extends Disposable {
 		this._register(AccessibleViewAction.addImplementation(100, 'panelChat', accessor => {
 			const accessibleViewService = accessor.get(IAccessibleViewService);
 			const widgetService = accessor.get(IChatWidgetService);
-			const widget: IChatWidget | undefined = widgetService.lastFocusedWidget;
-			const focusedItem: ChatTreeItem | undefined = widget?.getFocus();
+			const contextKeyService = accessor.get(IContextKeyService);
+			const commandService = accessor.get(ICommandService);
+			let widget: IChatWidget | undefined = widgetService.lastFocusedWidget;
+			let focusedItem: ChatTreeItem | undefined = widget?.getFocus();
+			let focusedInput = false;
+			if (!contextKeyService.getContextKeyValue(CONTEXT_RESPONSE.key)) {
+				// focus is in the input box, so we need to focus the last widget first
+				commandService.executeCommand('chat.action.focus');
```

Comment:
Can you do this without doing 'executeCommand' internally? I dislike this, it makes code hard to follow

Reply:
IDK, I could try to get the editor uri from the active editor like you do in chatActions.ts, but wasn't sure that would even work

Reply:
trying